scormxblock/scormxblock.py

```diff
@@ -431,7 +431,7 @@ def proxy_content(self, request, suffix=''):
                 content_type = mimetypes.types_map[ext]
         else:
             return Response('Did not exist in storage: ' + path_to_file + '\nstorage.path=' + storage.path(''), status=404, content_type='text/html', charset='UTF-8')
-        return Response(contents, content_type=content_type, headerlist=[('X-Frame-Options', 'SAMEORIGIN')])
```

Comment:
Are we sure that removing `same origin` header does not bring security leak in any use case?